docs/reference/announcements.md

```diff
@@ -50,6 +50,10 @@ The Zeebe Java client will not be developed further and will only receive bug fi
 - **Artifact ID change**:
   - The `artifactId` will change from `zeebe-client-java` to `camunda-client-java`.
 
+### Deprecation: Zeebe Go client & zbctl
+
+The Zeebe Go Client and zbctl will be officially deprecated with the 8.6 release as part of our efforts to streamline the Camunda 8 API experience. This client and cli utility will not get released with Camunda 8.6, will no longer receive new features, and will be transitioned to a community-maintained status.
```

Comment:
```suggestion
The Zeebe Go Client and zbctl will be officially deprecated with the 8.6 release as part of our efforts to streamline the Camunda 8 API experience. This client and CLI utility will not get released starting with Camunda 8.6, will no longer receive new features, and will be transitioned to a community-maintained status. 
```

Reply:
@mesellings are you fine with this as well?

Reply:
Yep, it's a good clarification - well spotted @aleksander-dytko

Reply:
Or we could reword to "will no longer get released"

Reply:
But either is fine.